### ISSUE TITLE
Allow recommended setting for a single Category

### ISSUE THREAD

**Issue by Cebrain** [undergroundwires/privacy.sexy#111]
I would really like a feature on the web page to set an template only for a category.
At the moment when i choose "Standard" all Cards get the settings. If i want only a configuration with a single category, i need to edit the checkboxes or file manually.

Would be great if we could "check" the categorys for which we want the settings.

**Comment by undergroundwires** [comment]
You're right. I need it as well, so I'll add this. So how about we place `Select: None | Standard | Strict | All` on top of each card? We could also add #68 (it's already implemented I'll push it after some changes) to revert all only inside a category. It would bloat the UI little bit but I cannot come up with better solution.

**Comment by undergroundwires** [comment]
Thanks for idea, I like it. I need it as well, so I'll add this. So how about we place `Select: None | Standard | Strict | All` on top of each card? We could also add #68 (it's already implemented I'll push it after some changes) to revert all only inside a category. It would bloat the UI little bit but I cannot come up with better solution.

**Comment by undergroundwires** [comment]
Thanks for idea, I like it. I need it as well, so I'll add this. So how about we place `Select: None | Standard | Strict | All` on top of each card? We could also add #68 (it's already implemented I'll push it after some more testing) to revert all only inside a category so I'll do this after that one is merged. It would bloat the UI little bit but I cannot come up with better solution.

**Comment by Cebrain** [comment]
This sounds good and is a fine solution :)

**Comment by marcello-pietrobon** [comment]
Has this been implemented and committed yet? the idea is excellent and very much needed.
Also the idea of a button (if I have well understood) to revert everything inside each individual category is equally needed.
